src/Microsoft.ML.Data/DebuggerExtensions.cs

```diff
@@ -45,7 +45,7 @@ public static DataDebuggerPreview Preview(this ITransformer transformer, IDataVi
             Contracts.CheckParam(maxRows >= 0, nameof(maxRows));
             Contracts.CheckParam(maxTrainingRows >= 0, nameof(maxTrainingRows));
 
-            var env = new LocalEnvironment(conc: 1);
+            var env = new LocalEnvironment();
```

Comment:
>LocalEnvironment [](start = 26, length = 16)

Do we have any tests for Preview method?
Can you double check it still works with you changes?

Reply:
We use it in some tests and I tried using it in the debugger and it seemed to be working.

---
In reply to: [262793799](https://github.com/dotnet/machinelearning/pull/2846#discussion_r262793799) [](ancestors = 262793799)